### PR TITLE
Jetpack Cloud: Add Monetize section

### PR DESCRIPTION
**Pull request by lupus2k**
Slack: p1711036278716469/1711029747.007349-slack-CKZHG0QCR

## Proposed Changes

We add `Monetize` section  to Jetpack Cloud.

## Known issues
- `Learn more` on Monetize subtitle links to WPcom support, Doc requested here: https://github.com/Automattic/jpop-quill/issues/158
- Privacy `Learn more` & `Privacy Information` links to WPcom support. Docs requested here: https://github.com/Automattic/jpop-quill/issues/167
- `Automattic Ads Terms of Service`  links to WPcom support. Docs requested here: https://github.com/Automattic/jpop-quill/issues/167
- Launchpad links to the wrong path, PR created (https://github.com/Automattic/jetpack/pull/36728) and should be deployed after we release Monetize on JP Cloud

## Tasks

- [x] Define icon for `Monetize` > Updated to `payment`
- [x] Should we maintain `/earn` or replace with `/monetize`? > Changed to `/monetize`
- [x] Check CSS. > CSS Updated.

#### Monetization Options

Atomic sites:
- [x] `Collect payments` & `Receive donations and tips` links to `https://wordpress.com/payments-donations/`. Maybe replace with `https://jetpack.com/support/jetpack-blocks/donations-block/`
- [x] `Create subscriber-only content` links to `https://wordpress.com/support/wordpress-editor/blocks/premium-content-block/`. Replaced it to `https://jetpack.com/support/jetpack-blocks/paid-content-block/`
- [x] `Set up a paid newsletter` links to `https://wordpress.com/support/paid-newsletters/`. Replaced with `https://jetpack.com/support/newsletter/paid-newsletters/`
- [x] `Collect PayPal payments` links to `https://wordpress.com/support/wordpress-editor/blocks/pay-with-paypal/`. Replaced with `https://jetpack.com/support/pay-with-paypal/`
- [x] `Refer a friend` has WPcom related copy and refer link is a WPcom link, removed from Self Hosted. 

Self-hosted sites:
- [x] `Collect payments`, `Receive donations and tips`, and `Collect PayPal payments`. Same as Atomic
- [x] `Earn ad revenue` with plan links to `/earn/ads-settings`. No plan links to `/pricing/` and adds `Learn more` that links to `wordads.co`

#### Supporters
- [x] `Learn more` links to `https://wordpress.com/support/wordpress-editor/blocks/payments/`. Replaced with `https://jetpack.com/support/jetpack-blocks/payments-block/`

#### Payment Settings
- [x] `Connect Stripe` opens an Stripe window but WPcom branded and back link redirects to WPcom. We can rebrand in a follow-up

## Testing Instructions

> [!NOTE]  
> To test the email design when joining WordAds, you have to apply this diff: D144101-code

* Run `yarn start-jetpack-cloud `
* Go to `http://jetpack.cloud.localhost:3000/monetize/:your-site`
* Check the copies.
* Check if support/help links to Jetpack support docs at Jetpack Cloud and WPcom docs on WPcom.
* Check if everything is working.
* Do it with an Atomic site and a self-hosted site.
* Check if nothing is broken on Calypso at `/earn/:your-site`

### CODE REVIEW

**Comment by crisbusquets** [comment]
Hi there!

I've checked Monetize in Jetpack Cloud and saw some issues:

Monetize-—-Jetpack-com


### Font size (red rectangle)
* The paragraph size should be the same as the heading (or smaller). After checking `/social`, I'd use font-size of 16 instead of 20.
* Mobile styles seem to be fine

image


### Grid (green lines)
The cards' grid width is narrower than the stats. They should be the same.

### Learn More buttons
Question: should we open them in a new tab?

* ❌ Collect Payments. It opens (https://jetpack.com/support/jetpack-blocks/donations-block/), and it should open (https://jetpack.com/support/jetpack-blocks/payments-block/)
* ✅ Donations and tips
* ✅ Subscriber-only content
* ✅ Paid newsletters
* ✅ PayPal
* ❌ Ads dashboard. I see a 404 error. The URL seems to be a Calypso/Earn one: https://container-musing-roentgen.calypso.live/earn/ads-earnings/jetpackgoku.wpcomstaging.com
* ❌ Refer a friend. Should we remove this for Jetpack? (cc @simison )


### Monetize > Payment Settings
The `Upgrade to lower it` link opens the Jetpack pricing page. I understand why, but the path its not clear. Upgrade... what? **Should we open the Checkout with the Creator plan?**

Screenshot 2024-03-27 at 17 38 53

**Comment by simison** [comment]
> Refer a friend. Should we remove this for Jetpack?

Yep, it's not visible for Jetpack sites on Calypso blue version either.

**Comment by lupus2k** [comment]
> ### Font size (red rectangle)
> * The paragraph size should be the same as the heading (or smaller). After checking `/social`, I'd use font-size of 16 instead of 20.

Fixed

> ### Grid (green lines)
> The cards' grid width is narrower than the stats. They should be the same.

Done

> ### Learn More buttons
> * ❌ Collect Payments. It opens (https://jetpack.com/support/jetpack-blocks/donations-block/), and it should open (https://jetpack.com/support/jetpack-blocks/payments-block/)

Fixed

> * ❌ Ads dashboard. I see a 404 error. The URL seems to be a Calypso/Earn one: https://container-musing-roentgen.calypso.live/earn/ads-earnings/jetpackgoku.wpcomstaging.com

Fixed

> * ❌ Refer a friend. Should we remove this for Jetpack? (cc @simison )

Done, only show for WPcom sites

> ### Monetize > Payment Settings
> The `Upgrade to lower it` link opens the Jetpack pricing page. I understand why, but the path its not clear. Upgrade... what? **Should we open the Checkout with the Creator plan?**

We can do this for WPcom sites, or maybe linking to WPcom plans page as Calypso's monetize is doing now.
Should we keep linking JP Sites to `/pricing/`? I see no commission data there :thinking: 

> Question: should we open them in a new tab?

Not sure about this... cc @simison

**Comment by simison** [comment]
> We can do this for WPcom sites, or maybe linking to WPcom plans page as Calypso's monetize is doing now.
> Should we keep linking JP Sites to /pricing/? I see no commission data there 🤔

WP.com sites need to continue linking to WP.com pricing page inside calypso for that site.

For Jetpack sites, good question! @mikestottuk what's the best place to link for an upsell to lower % for paid newsletters?

**Comment by simison** [comment]
> should we open them in a new tab?

@crisbusquets I'm a bit vary of opening new tabs unless we're afraid to lose content/settings they're editing, which isn't the case here. New tabs can be opened by users manually in different ways if they need, browser back buttons work well for these pages, and new tabs tend to get people lost in mobile phones.

Not a strong opinion, but let's keep them as same-page now.

**Comment by crisbusquets** [comment]
> Not a strong opinion, but let's keep them as same-page now.

Works for me!

**Comment by lupus2k** [comment]
WordAds email templates:
| WordPress | Jetpack | 
| --- | --- |
| wordads-email | wordads-jetpack |

Should we update the header, colors and email address to match what we did on Subscribers here (D141993-code) ?

File: client/my-sites/earn/components/commission-fees.tsx [hunk 1]
```diff
@@ -18,16 +22,27 @@ const CommissionFees = ( {
...
+	const upgradeLinkHost = isJetpackNotAtomic ? 'https://jetpack.com' : 'https://wordpress.com';
+
 	const upgradeLink =
 		commission === 0 || isPlan100YearPlan ? null : (
 			<>
 				{ ' ' }
-				<a href={ siteSlug ? `/plans/${ siteSlug }` : '/plans' }>
+				<a
+					href={
+						siteSlug ? `${ upgradeLinkHost }/plans/${ siteSlug }` : `${ upgradeLinkHost }/plans`
```
Comment (candy02058912) [inline client/my-sites/earn/components/commission-fees.tsx]: This breaks for me on a Jetpack self hosted site, for example I had a site: `shadowy-post-tree.jurassic.ninja` so it links to https://jetpack.com/plans/shadowy-post-tree.jurassic.ninja which shows Page not found.

**Comment by candy02058912** [comment]
> Should we update the header, colors and email address to match what we did on Subscribers here

I vote for yes 🎉

File: client/my-sites/earn/components/commission-fees.tsx (hunk 1 above)
Comment (lupus2k) [reply client/my-sites/earn/components/commission-fees.tsx]: Thanks for noticing this. I fixed it, but we are waiting for an answer from here: https://github.com/Automattic/wp-calypso/pull/88778#issuecomment-2024819919

Comment (lupus2k) [reply client/my-sites/earn/components/commission-fees.tsx]: We have an answer and I updated the link to https://jetpack.com/creator/#pricing

**Comment by allilevine** [comment]
These two links 404 for me:
Screen Shot 2024-04-03 at 4 24 07 PM

Should the URL be /monetize/ instead of /earn/?

**Comment by allilevine** [comment]
Where should "Learn more" link to? 
Screen Shot 2024-04-03 at 4 30 11 PM

**Comment by allilevine** [comment]
This info pop-up has links to WordPress.com:

Screen Shot 2024-04-03 at 4 39 10 PM

**Comment by lupus2k** [comment]
> Where should "Learn more" link to?

Asked for support doc update: https://github.com/Automattic/jpop-quill/issues/158

> This info pop-up has links to WordPress.com:

Asked for support doc here: https://github.com/Automattic/jpop-quill/issues/167

**Comment by lupus2k** [comment]
> These two links 404 for me: Screen Shot 2024-04-03 at 4 24 07 PM
> 
> Should the URL be /monetize/ instead of /earn/?

I can't see this screen, how did you get it? :thinking:

**Comment by allilevine** [comment]
> I can't see this screen, how did you get it? 🤔

It's at the top of /monetize for me:
Screen Shot 2024-04-03 at 5 01 04 PM

I should've taken a bigger screenshot. 🙂

**Comment by foleynotrose** [comment]
@lupus2k :

> Asked for support doc update: https://github.com/Automattic/jpop-quill/issues/158

I've in turn asked for some help on this from @mikestottuk or @simison. 

> https://github.com/Automattic/jpop-quill/issues/167

I'll take care of this one.

**Pull request merged**